assets/js/profile.js

```diff
@@ -3,13 +3,19 @@ const userNameEle = document.getElementById("userName");
 const navigateEle = document.getElementById("navigate-user-text");
 
 // Navigate to practice.html if no data in local storage
-const checkIfUserData = () => {
+const checkIfUserData = (msg) => {
     const name = localStorage.getItem("typerName");
     if(!name){
         const linktoprofile = document.createElement('a');
         linktoprofile.innerText = "Click here";
         linktoprofile.href = "./practice.html";
-        navigateEle.innerText = " to set up your profile";
+        navigateEle.innerText = `${msg}`;
+        navigateEle.insertAdjacentElement('afterbegin', linktoprofile);
+    }else{
+        const linktoprofile = document.createElement('a');
+        linktoprofile.innerText = "Click here";
+        linktoprofile.href = "./practice.html";
+        navigateEle.innerText = "to play again";
```

Comment:
Change `to play again` to `to practice again`.

Reply:
ok